mne/tests/test_dipole.py

```diff
@@ -140,6 +141,13 @@ def test_dipole_fitting(tmpdir):
                                    rank='info')  # just to test rank support
     assert isinstance(residual, Evoked)
 
+    # Test conversion of dip.pos to MNI coordinates.
+    dip_mni_pos = dip.to_mni('sample', fwd['mri_head_t'],
+                             subjects_dir=subjects_dir)
+    head_to_mni_dip_pos = head_to_mni(dip.pos, 'sample', fwd['mri_head_t'],
```

Comment:
can you try making one call with the str input passing the file name? find the -trans.fif file in the testing data.
just to make sure we test the different input types